sessions/recap-project-4/example-implementation/src/App.js

```diff
@@ -0,0 +1,81 @@
+import useLocalStorageState from "use-local-storage-state";
+import { useEffect } from "react";
+import "./App.css";
+import Form from "./components/Form/index.js";
+import List from "./components/List/index.js";
+import { nanoid } from "nanoid";
+
+function App() {
+  const [activities, setActivities] = useLocalStorageState("activities", {
+    defaultValue: [],
+  });
+  const [weather, setWeather] = useLocalStorageState("weather", {
+    defaultValue: null,
+  });
+  console.log(weather);
+
+  useEffect(() => {
+    async function fetchWeather() {
+      try {
+        const response = await fetch(
+          "https://example-apis.vercel.app/api/weather"
```

Comment:
I am not sure: is this still an issue or not?

I don't know the exact conditions for good / bad weather, so I think it's rather something to change in the API itself.

Reply:
I'm assuming the temperature automatically increases and decreases? I can't tell you whether the logic/functionality of the API being fairly opaque is an issue or not. This is more of a (half-serious) observation rather than a complaint. 🤷‍♀️ I'll ask @djfarly 😂

Reply:
@djfarly is on the job :)